crates/turborepo-auth/src/auth/login.rs

```diff
@@ -27,26 +27,30 @@ pub async fn login<T: Client + TokenClient>(options: &LoginOptions<'_, T>) -> Re
         ui,
         login_url: login_url_configuration,
         login_server,
-        sso_team: _,
         existing_token,
+        force,
+        sso_team: _,
     } = *options; // Deref or we get double references for each of these
 
     // Check if passed in token exists first.
-    if let Some(token) = existing_token {
-        if Token::existing(token.to_string())
-            .is_valid(api_client)
-            .await?
-        {
-            return check_user_token(token, ui, api_client, "Existing token found!").await;
+    if !force {
```

Comment:
I get really confused by negative if blocks, and almost always feel like they should be inverted. Not blocking at all, but if you feel like restructuring, something like this could be nice:

```rs
if force {
  return login()
}

if let Some(token) = existing_token {
  return "yay";
}

login();
```

Reply:
I'm down to do something like this, but I'll make it in another stack because this'll bork / change up the tests